### PR TITLE
otlp_config_linux.yaml - add otlphttp for metrics

### PR DESCRIPTION
**Pull request by Kielek**
**Description:** Add example how to send metrics over otlphttp

**Link to Splunk idea:** Based on created example for https://github.com/signalfx/tracing-examples/pull/169

**Testing:** Tested in https://github.com/signalfx/tracing-examples/pull/169

**Documentation:** N/A?

### CODE REVIEW

File: cmd/otelcol/config/collector/otlp_config_linux.yaml [hunk 1]
```diff
@@ -117,7 +118,7 @@ service:
     metrics:
       receivers: [otlp, signalfx, smartagent/signalfx-forwarder]
       processors: [memory_limiter, batch]
-      exporters: [signalfx]
+      exporters: [otlphttp, signalfx]
```
Comment (Kielek) [inline cmd/otelcol/config/collector/otlp_config_linux.yaml:121]: The question is if here we need both exporters. Same for traces.

Comment (rmfitzpatrick) [reply cmd/otelcol/config/collector/otlp_config_linux.yaml:121]: The signalfx exporter has a number of built in translations that work best w/ existing built-in content. You are also now adding double publishes.

**Review by rmfitzpatrick — CHANGES_REQUESTED** [review]
This config is for use and not just example. If the ask is to document how to use the otlp http exporter, I'd suggest updating a readme or product doc.

**Comment by theletterf** [comment]
I think this could be the doc that could host the PR's contents: https://docs.splunk.com/observability/en/gdi/opentelemetry/components/otlphttp-exporter.html

**Pull request closed without merging**